aodndata/moorings/products_handler.py

```diff
@@ -310,9 +344,15 @@ def preprocess(self):
 
         if 'aggregated' in self.products_to_create:
             self._make_aggregated_timeseries()
-        if 'hourly' in self.products_to_create:
+
```

Comment:
Maybe a good idea is to explicitly set the dependencies instead of using top-bottom separated if:
```
gridded_product_mlist = [self.make_aggregated_timeseries, self.make_hourly_timeseries, self.make_gridded_timeseries ]
hourly_product_mlist = [...]
...
if 'gridded' in self.products_to_create:
clist = gridded_product_mlist
elif  ...
...
for create_product in clist:
     self.logger.info(...)
     create_product()